### ISSUE TITLE
How to create Prompt?

### ISSUE THREAD

**Issue by saisusmitha** [xuxuxuxuxuxjh/LB-UNet#1]
In dataset.py- you have mentioned points_list = os.listdir(path_Data+'train/points_boundary2/')- where are these available? how to create them? Does boundary.py create these? In boundary.py there is no creation of such files. I am confused kindly let me know how to create prompts.

**Comment by xuxuxuxuxuxjh** [comment]
Yes, you can use boundary.py to create boundary maps. And you can place these maps wherever, as long as they correspond to the directory names in dataset.py.

**Comment by saisusmitha** [comment]
Thanks for replying, I already did that 
1) I ran your code exactly and best_val is being printed but no png is being saved. So I thought  there is no line creating the output_path2 directory in the code

2) So I just added a line "Path(output_path2).mkdir(parents=True, exist_ok=True)" to create the folder- then images were saved into the  output_path2 folder - they were taking a lot of time but images are being saved - they look empty - completely black and around 650-670 bytes that's it.

Can you please tell me why this is happening and show me a sample image saved by boundary.py showing what it looks like? what exactly the boundary.py is saving? 
 

![ISIC_0000024_segmentation](https://github.com/user-attachments/assets/dac3d745-eeaa-4c55-bb49-d41673a47746)
![ISIC_0000024_segmentation](https://github.com/user-attachments/assets/dee31b3b-9784-49ee-89fb-3ab5151e2a16)

**Comment by xuxuxuxuxuxjh** [comment]
![image](https://github.com/user-attachments/assets/cc68a9e9-aa39-451f-93d6-ee623ee140e7)
![image](https://github.com/user-attachments/assets/d6ed0e36-2c86-4ab5-8f53-95a12dba0b43)

**Comment by xuxuxuxuxuxjh** [comment]
Just motified the data path, the model will run well. But if your data images are not skin lesion images, maybe having more than two connected blocks, the model will fail.

**Comment by xuxuxuxuxuxjh** [comment]
Just modify the data path, and then the model will run well. But if your data images are not skin lesion images, maybe having more than two connected blocks, the model will fail.

**Comment by saisusmitha** [comment]
Thanks, I am using the ISIC2017 and 2018 data,  but I don't understand why i am getting blank outputs. 

1) Can you share the boundary folder/images with me (arvapallisaisusmitha@gmail.com) , drive link works
and
2) Can you please crosscheck if the uploaded boundary.py is the same code you used above?

**Comment by xuxuxuxuxuxjh** [comment]
I have sent the folder to your email, and I assure you that the uploaded boundary.py is the same code I used.

**Comment by hz1z** [comment]
@xuxuxuxuxuxjh  Hello ,Can you share the boundary folder/images with me ([2576642465@qq.com](2576642465@qq.com)) or [houyuanhao2@gmail.com](houyuanhao2@gmail.com) too .

**Comment by shushuang010313** [comment]
Hello, I set the hyperparameters according to the paper, but the loss is NaN. How can I solve this problem? 
![2024111105250](https://github.com/user-attachments/assets/720184cc-a348-4cd3-a22f-971edfc09b93)
![2024111105418](https://github.com/user-attachments/assets/c05f6cbe-9596-4cd3-b3fe-d91faa61fe9e)

**Comment by xuxuxuxuxuxjh** [comment]
There is something wrong. I can't see the pictures.

**Comment by rrbsa** [comment]
@xuxuxuxuxuxjh Hello ,Can you share the boundary folder/images with me ([825473554@qq.com] too .
